### ISSUE TITLE
Daily Build / Beta Icon for all OS

### ISSUE THREAD

**Issue by dagar** [mavlink/qgroundcontrol#5511]
In the Play store the QGroundControl and QGC (Daily Test Build) have slightly different icons, but on the active device they're using the same icon.

![image](https://user-images.githubusercontent.com/84712/28755018-d887e050-751f-11e7-9312-85cd363070cb.png)

We need an a better beta/test icon generated for android in all sizes, then build scripting to change the icon here. https://github.com/mavlink/qgroundcontrol/blob/master/android/AndroidManifest.xml#L3
https://github.com/mavlink/qgroundcontrol/blob/master/tools/update_android_version.sh#L16

**Comment by hamishwillee** [comment]
The UX team at Intel are going to love the opportunity to help with this. BUT if we want help we need to provide very clear direction. So, list of what you want ...

**Comment by dagar** [comment]
Something similar to what Google did with Android Chrome?

![image](https://user-images.githubusercontent.com/84712/28758155-66264320-7561-11e7-8dfa-60dee6bf7886.png) https://play.google.com/store/search?q=chrome

**Comment by hamishwillee** [comment]
Also, are we happy with the current iconography, or tempted by elements of the proposed DC branding: https://www.youtube.com/watch?v=i4kblPyLdBw&feature=youtu.be

W.r.t. the above @dagar I like that, but this exact approach might cut the bottom off the Q. All we want is something that is recognizably QGroundControl and says "Beta" right?

**Comment by anassinator** [comment]
@dagar, @hamishwillee:

From my experience dogfooding, I've seen two ways of distinguishing the beta/nightly builds from prod:
- Adding a banner as Chrome does above
- Changing the colors or background somehow to make it look like a WIP or different enough from prod such that they can't be confused

What do you guys think of changing the background instead? This is a quick and rough example of what I mean:

![android_dev_512x512](https://user-images.githubusercontent.com/723610/28808287-6dd78728-7683-11e7-8f7a-62766994bb99.png)

Original for comparison:

![android_512x512](https://user-images.githubusercontent.com/723610/28808130-6b6edcb2-7682-11e7-9dd9-83808cfd85e2.png)

On a side note, I'd love to see a more DC-inspired icon.

**Comment by dagar** [comment]
I'm not sure if the default QGC coloring is sufficiently recognizable at this point. Can you still slap a beta label across it?

**Comment by DonLakeFlyer** [comment]
like the graph paper style thing. But agree that some sort of Beta label is likely still needed.

**Comment by NaterGator** [comment]
I don't have an artistic bone in my body or I would mock this up, but could you stylize the "Q" portion of the beta logo to look like a sketch / draft of the letter's geometry (roughly this premise: http://imgur.com/a/c1Kiu)

**Comment by hamishwillee** [comment]
@NaterGator Looks cool - but IMO a bit far from the original :-) Lets get the UXWG team to do the actual work of producing the logo. What we need to do is provide them with ideas on what we think might work/we could live with. 

I think we all want "Beta" on the logo. We all want this to remain clearly "QGroundControl". We like the graph paper thing, but open to other ideas. Reasonable feedback to give them?

**Comment by anassinator** [comment]
@NaterGator I think the sketch could be a neat idea, but don't have the time to whip that up right now.
@hamishwillee I think he meant we could have the entire icon be the same as the original but to look completely hand-drawn on a blueprint or something instead. Correct me if I'm wrong.

I am quickly putting the following up simply for a better idea of what this could look like.

Blueprint, bottom label:
![android_512x512_bottom](https://user-images.githubusercontent.com/723610/28873884-1baf2f94-7798-11e7-9f03-e52137726d0c.png)

Blueprint, corner label:
![android_dev_512x512_corner](https://user-images.githubusercontent.com/723610/28873895-26a0e0fa-7798-11e7-90d8-1d0c5ae1ced6.png)

Original, bottom label:
![android_512x512_bottom_noblueprint](https://user-images.githubusercontent.com/723610/28873947-69b66c20-7798-11e7-9cf2-e3b67f369124.png)

Original, corner label:
![android_512x512_corner_noblueprint](https://user-images.githubusercontent.com/723610/28873951-6e054076-7798-11e7-9791-4f39b0844b09.png)

The corner label can obviously be on any corner. I also didn't put any thought into the colors, fonts, sizing, background and etc. I'm not a designer and I'm certain the UXWG team can do a better job; I'm just trying to help out so that you may have something more concrete to give.

Also, this probably goes without saying, but I think we should do the same for the desktop apps and iOS (not sure if the TestFlight is still a thing).

**Comment by NaterGator** [comment]
Yeah, I meant basically composite what looks like sketching the geometry of the Q in to give it a "work in progress" feel. To be honest 1 and 2 look pretty great and are a quick glance to realize this is not the stable release branch.

**Comment by hamishwillee** [comment]
Yes, they (https://github.com/anassinator) all look really good to me. I slightly prefer the graph background, and the "corner" orientation. If we're all basically OK, I can forward this to UXWG?

>  think we should do the same for the desktop apps and iOS (not sure if the TestFlight is still a thing).

So I was assuming we'd use same Beta icon for all platforms. Is that what you're saying?

FYI I did understand what @NaterGator was suggesting. I don't hate the idea. To me though it felt more like something to capture a prototype :-) than a beta.

**Comment by anassinator** [comment]
@hamishwillee 

Yeah it should be the same design, but each platform has different sizes and formats requirements. For example, the iOS icon must not be rounded as the OS adds that itself.

Also, I would suggest either reworking the rounding radius of the icon for each platform as it seems too harsh right now. I haven't been following Android very much lately but I think I've seen a lot of completely circular (like chrome above) and completely square icons. And for example Mac or has a lot more circular icons as well. So, maybe we should investigate shapes that better suite each platform. This can also affect whether corner or bottom makes more sense.

And ok was just making sure we're on the same page 🙂

Finally, moving forward with UXWG isn't my call, but I'm OK with that 😛

**Comment by DonLakeFlyer** [comment]
I'd rather not have different branding on each platform. Also I'd rather not change the existing branding (other than a beta version) at this point. You bounce around too much and people don't know what's what.

**Comment by hamishwillee** [comment]
@DonLakeFlyer Thanks for that Don. I'll see whether UXWG care to comment. If they don't come back with clear guidance or a plan for clear guidance in a week, I personally would be happy with us all to make an executive decision :-) Perhaps on option 2 [above](https://github.com/mavlink/qgroundcontrol/issues/5511#issuecomment-319661447)

**Comment by DonLakeFlyer** [comment]
I think this one is fine:
![28873895-26a0e0fa-7798-11e7-90d8-1d0c5ae1ced6](https://user-images.githubusercontent.com/5876851/28904402-88fef6c0-77bf-11e7-82e3-22178aad9df6.png)

**Comment by hamishwillee** [comment]
Me to. Just sent an email to UXWG for comment, listing that as my preference and asking for comment/professional versions in the right sizes. Lets give them a few days to respond - OK?

**Comment by anassinator** [comment]
@hamishwillee, @DonLakeFlyer:

That's good by me.

For reference, since I didn't see these comments in time, this is what I meant about a circular icon:

![comparison](https://user-images.githubusercontent.com/723610/28904679-52c815ae-7815-11e7-924e-458499a462c9.png)

iOS would need to remain a square, but IMO this isn't changing the branding.

**Comment by DonLakeFlyer** [comment]
> but IMO this isn't changing the branding.

The square with rounded edges are definitely part of the banding. It's used in many places.

**Comment by anassinator** [comment]
Ah I see what you mean. Fair enough 👍

**Comment by DonLakeFlyer** [comment]
Change title since we want to do this for all OS.

**Comment by fabricionovak** [comment]
The blueprint background and Beta tag will do the job. My only concern is about legibility in different sizes. I've made some tweaks and tests.

![scene](https://user-images.githubusercontent.com/4181566/28938732-cf015c6a-7843-11e7-8247-ef88023003f4.png)

![192x192](https://user-images.githubusercontent.com/4181566/28938734-cf13b13a-7843-11e7-9934-0605543cf817.png)

![qgc_beta_icon_homescreen](https://user-images.githubusercontent.com/4181566/28938733-cf01fe04-7843-11e7-9b60-e2c561f1a4b3.png)

![android_icon_sizes_scale](https://user-images.githubusercontent.com/4181566/28938731-cefe9bf6-7843-11e7-93a3-b5d4e2723450.png)

**Comment by anassinator** [comment]
That looks nice.
Yeah the beta label might be hard to read on lower resolution devices. That's why a noticeably different background is common and would probably be the best indicator IMO. We can still have the label as well.

**Comment by fabricionovak** [comment]
@anassinator I'd avoid changing the background color. The color combination is part of the branding guidelines. The beta label is small, but it is pretty readable in my opinion. Here is a picture of my phone right now (not a mockup like the previous one).

![img_20170803_130331](https://user-images.githubusercontent.com/4181566/28941524-27e97890-784d-11e7-89ff-61977870d28d.jpg)

**Comment by anassinator** [comment]
@fabricionovak:
Oh I thought you were worried about lower DPI devices where the label might be much harder to make out.
As for the branding issue, I understand the concern, and it's up to you guys for sure. I prefer the one you posted anyways :)

**Comment by fabricionovak** [comment]
These lower DPI devices will not display the text as sharp as the high-end ones, but it will be still readable.

**Comment by hamishwillee** [comment]
OK, @fabricionovak I think we'll all be happy with this version you have proposed. Can we have versions suitable for desktop, android and iOS? [Do you need list of specific sizes/formats?]

**Comment by fabricionovak** [comment]
@hamishwillee a list would be helpful. :)

**Comment by hamishwillee** [comment]
@fabricionovak The android icon requirements are here: https://developer.android.com/guide/practices/ui_guidelines/icon_design_launcher.html#size
(note the need for the android play store icon in the following section).

iOS app icon sizes are here: https://developer.apple.com/ios/human-interface-guidelines/graphics/app-icon/

@DonLakeFlyer @dagar Is that correct set? What sizes are needed for desktop?

**Comment by hamishwillee** [comment]
@fabricionovak I know you're busy. Can you please ETA some of these icons?

**Comment by fabricionovak** [comment]
@hamishwillee I'll work on this today.

**Comment by hamishwillee** [comment]
Thanks @fabricionovak  !

**Comment by fabricionovak** [comment]
Android icons in PNG. Still working on the iOS versions

![ic_qgc_beta_android_google_play](https://user-images.githubusercontent.com/4181566/29146735-c2009cc8-7d17-11e7-914c-f7c5a6a1d0e4.png)
![ic_qgc_beta_android_hdpi](https://user-images.githubusercontent.com/4181566/29146734-c1ffc938-7d17-11e7-857c-98e6c55ffca8.png)
![ic_qgc_beta_android_mdpi](https://user-images.githubusercontent.com/4181566/29146737-c2026e0e-7d17-11e7-8b26-258faa028c76.png)
![ic_qgc_beta_android_xhdpi](https://user-images.githubusercontent.com/4181566/29146733-c1fdfb80-7d17-11e7-8fd2-42f75618c037.png)
![ic_qgc_beta_android_xxhdpi](https://user-images.githubusercontent.com/4181566/29146736-c200ee30-7d17-11e7-99e3-7c5a6cf7ff0c.png)
![ic_qgc_beta_android_xxxhdpi](https://user-images.githubusercontent.com/4181566/29146738-c20463ee-7d17-11e7-8a2b-357d9a57a646.png)

**Comment by fabricionovak** [comment]
PSDs for android

[ic_qgc_beta_android.zip](https://github.com/mavlink/qgroundcontrol/files/1213061/ic_qgc_beta_android.zip)

**Comment by hamishwillee** [comment]
Thank Fabricio! @dagar @DonLakeFlyer  What else do you need/can you take it from here?

**Comment by DonLakeFlyer** [comment]
I'll get going on this. I'll yell if I need antyhing.

**Comment by fabricionovak** [comment]
PNG files for iOS devices
![ic_qgc_beta_appstore](https://user-images.githubusercontent.com/4181566/29293446-080bda34-8100-11e7-8775-0c09c8d63c76.png)
![ic_qgc_beta_ipad_ 2x](https://user-images.githubusercontent.com/4181566/29293447-080d0634-8100-11e7-8a14-03cf588c2d92.png)
![ic_qgc_beta_ipadpro_ 2x](https://user-images.githubusercontent.com/4181566/29293450-080feb60-8100-11e7-8aa2-aa170963a247.png)
![ic_qgc_beta_iphone_ 2x](https://user-images.githubusercontent.com/4181566/29293449-080ec078-8100-11e7-83d8-ddc119068290.png)
![ic_qgc_beta_iphone_ 3x](https://user-images.githubusercontent.com/4181566/29293448-080df67a-8100-11e7-9949-8ad7f4e7e777.png)

**Comment by fabricionovak** [comment]
iOS icons in PSD
[ic_qgc_beta_iOS.zip](https://github.com/mavlink/qgroundcontrol/files/1223560/ic_qgc_beta_iOS.zip)

**Comment by fabricionovak** [comment]
Android icon update in XXXHDPI. Please, replace the previous files (ic_qgc_beta_android_XXXHDPI.psd and ic_qgc_beta_android_XXXHDPI.png) for these.

![ic_qgc_beta_android_xxxhdpi](https://user-images.githubusercontent.com/4181566/29293693-cab7d20e-8100-11e7-8302-8443bac732bf.png)
[ic_qgc_beta_android_XXXHDPI.zip](https://github.com/mavlink/qgroundcontrol/files/1223566/ic_qgc_beta_android_XXXHDPI.zip)

**Comment by DonLakeFlyer** [comment]
Won't be closed until their use is implemented.

**Comment by DonLakeFlyer** [comment]
I started hooking these up, but they won't go in until 3.3 is out the door. Then dev/daily builds will cut over to these.

**Comment by Prasad28-97** [comment]
Hii Guys 
I want to change the default icon of QGC in my project. Can you tell me exactly where i need to make changes?

Thanks
